tests/test_handlers.py

```diff
@@ -456,6 +456,34 @@ def test_am1bcc_parameterization():
     # assert vjp_fn(charges_adjoints) == None
 
 
+def test_resp_parameterization():
+    # currently takes no parameters
+    smirks = []
+    params = []
+    props = None
+
+    cache_key = nonbonded.RESP
+    am1h = nonbonded.RESPHandler(smirks, params, props)
+    mol = Chem.AddHs(Chem.MolFromSmiles("C1CNCOC1F"))
+    AllChem.EmbedMolecule(mol)
```

Comment:
If RESP to be independent of the initial conformer, we should explicitly be calling `mol.RemoveAllConformers()` beforehand as opposed to `EmbedMolecule()`